Readme.md

```diff
@@ -89,6 +89,30 @@ or
 MDBOOK_preprocessor__graphviz__output_to_file="true" mdbook build
 ```
 
+### Embedding dot files
+Sometimes you don't want to write dot code, but instead include it from a file:
+
+~~~markdown
+```dot
+{{#include path/to/file.dot}}
+```
+~~~
+
+In this case, you might want to modify the order of preprocessors, so the
+include directives get resolved before it's passed to Graphviz.
+
+In that case, make sure your `[preprocessor.graphviz]` section in the config
+orders itself after `links`:
+
+```toml
+[preprocessor.graphviz]
+after = ["links"]
+```
+
+More information about preprocessors and ordering can be found
+[here](https://rust-lang.github.io/mdBook/format/configuration/preprocessors.html?highlight=preprocessors#require-a-certain-order).
+
+
 ### .gitignore
```

Comment:
This `.gitignore` section is part of the `Output To File` section. Want to move `Embedding dot files` above or below and give it a 2 `##` heading to line up with the others?

Reply:
I moved it to the end and made it a 2 `##` heading.